docs/01-app/02-guides/mdx.mdx

```diff
@@ -79,18 +79,18 @@ Create an `mdx-components.tsx` (or `.js`) file in the root of your project to de
 ```tsx filename="mdx-components.tsx" switcher
 import type { MDXComponents } from 'mdx/types'
 
-export function useMDXComponents(components: MDXComponents): MDXComponents {
-  return {
-    ...components,
-  }
+const components = {} satisfies MDXComponents
```

Comment:
Let's go with what we have further down too, or vice versa, but let's stick to one

```suggestion
const components: MDXComponents = {}
```